decidim-core/app/cells/decidim/redesigned_author_cell.rb

```diff
@@ -118,8 +118,13 @@ def creation_date?
     end
 
     def creation_date
-      date_at = from_context.try(:published_at) || from_context.try(:created_at)
-
+      date_at = if from_context.respond_to?(:published_at) &&
```

Comment:
Why do you want to change this method like this?

The previous version is more readable and I don't understand why we would need to change it from the original.